### PR TITLE
Added Br as a supported element

### PR DESCRIPTION
**Pull request by alongd**
### Motivation or Problem
RMG currently supports F, Cl, I, This PR adds support for another halogen, Br.

### Description of Changes
Updated atom types (plus docs), Reaction.get_sorting_key() and the Chemkin element list.

### Testing
Added an atom types test, also added to isomorphism test

### Reviewer Tips
Make sure all tests pass, see that the code makes sense.

### CODE REVIEW

File: rmgpy/chemkin.pyx [hunk 1]
```diff
@@ -2337,7 +2337,8 @@ def write_elements_section(f):
     s = 'ELEMENTS\n'
 
     # map of isotope elements with chemkin-compatible element representation:
-    elements = ('H', ('H', 2), ('H', 3), 'C', ('C', 13), 'O', ('O', 18), 'N', 'Ne', 'Ar', 'He', 'Si', 'S', 'Cl', 'I')
+    elements = ('H', ('H', 2), ('H', 3), 'C', ('C', 13), 'O', ('O', 18), 'N', 'Ne', 'Ar', 'He', 'Si', 'S',
+                'Cl', 'Br', 'I')
```
Comment (davidfarinajr) [inline rmgpy/chemkin.pyx]: I think we should also add `F` to this

**Comment by alongd** [comment]
thanks, @davidfarinajr, the comments were addressed in fixup commits

**Comment by alongd** [comment]
@sarahkha, @amarkpayne, any comments, or can I squash and rebase?

File: rmgpy/molecule/atomtypeTest.py [hunk 2]
```diff
@@ -483,26 +483,29 @@ def setUp(self):
...
-                                                     2 H u0 p0 c0 {1,S}
-                                                     3 H u0 p0 c0 {1,S}
-                                                     4 X u0 p0 c0
-                                                     5 H u0 p0 c0 {1,S}
-                                                     6 H u0 p0 c0 {1,S}''')
+                                                       2 H u0 p0 c0 {1,S}
+                                                       3 H u0 p0 c0 {1,S}
+                                                       4 X u0 p0 c0
+                                                       5 H u0 p0 c0 {1,S}
+                                                       6 H u0 p0 c0 {1,S}''')
```
Comment (amarkpayne) [inline rmgpy/molecule/atomtypeTest.py]: When you squash, put these whitespace changes in a separate "Minor:" commit

File: rmgpy/molecule/element.py [hunk 3]
```diff
@@ -120,16 +120,16 @@ class PeriodicSystem(object):
     `lone_pairs`: the number of lone pairs an element has
     'electronegativity': The atom's electronegativity (how well can an atom attract electrons), taken from
     https://sciencenotes.org/list-of-electronegativity-values-of-the-elements/
-    isotops of the same element may have slight different electronegativities, which is not reflected below
+    isotopes of the same element may have slight different electronegativities, which is not reflected below
```
Comment (amarkpayne) [inline rmgpy/molecule/element.py]: Include this in the "Minor:" commit

File: rmgpy/molecule/atomtype.py [hunk 4]
```diff
@@ -672,10 +672,10 @@ def get_features(self):
 ATOMTYPES['F'].set_actions(increment_bond=[], decrement_bond=[], form_bond=['F'], break_bond=['F'], increment_radical=['F'], decrement_radical=['F'], increment_lone_pair=[], decrement_lone_pair=[])
 ATOMTYPES['F1s'].set_actions(increment_bond=[], decrement_bond=[], form_bond=['F1s'], break_bond=['F1s'], increment_radical=['F1s'], decrement_radical=['F1s'], increment_lone_pair=[], decrement_lone_pair=[])
 
-#these are ordered on priority of picking if we encounter a more general atomtype for make
-allElements=['H', 'C', 'O', 'N', 'S', 'Si', 'Cl', 'F', 'I', 'Ne', 'Ar', 'He', 'X']
-#list of elements that do not have more specific atomTypes
-nonSpecifics=['H', 'He', 'Ne', 'Ar',]
+# these are ordered in priority of picking if a more general atomtype is encountered
```
Comment (amarkpayne) [inline rmgpy/molecule/atomtype.py]: Include in "Minor:" as well

File: rmgpy/molecule/atomtype.py [hunk 5]
```diff
@@ -672,10 +672,10 @@ def get_features(self):
...
+allElements = ['H', 'C', 'O', 'N', 'S', 'Si', 'F', 'Cl', 'Br', 'I', 'Ne', 'Ar', 'He', 'X']
+# list of elements that do not have more specific atomTypes
+nonSpecifics = ['H', 'He', 'Ne', 'Ar',]
```
Comment (amarkpayne) [inline rmgpy/molecule/atomtype.py]: Minor commit as well

**Comment by alongd** [comment]
Thanks for the feedback, rebased and squashed!

**Pull request merged**